src/__tests__/actions.spec.ts

```diff
@@ -110,6 +113,30 @@ describe('Code Editor Shortcuts: actions', () => {
       expect(cursor.line).toEqual(1);
     });
 
+    it('should delete to the end of the line', () => {
+      editor.setCursor({ line: 1, ch: 1 });
+      deleteToEndOfLine(editor as any);
+
+      const { doc } = getDocumentAndSelection(editor);
+      expect(doc).toEqual('lorem ipsum\nd\namet');
+    });
+
+    it('should delete the newline when at the end of the line', () => {
+      editor.setCursor({ line: 1, ch: 10 });
```

Comment:
Should this be `ch: 9` to more accurately capture 'end of line'? (Since position 10 is past the end of the text)

```
dolor sit
0123456789
```

Reply:
That's absolutely right, I've fixed that up.